lib/common.sh

```diff
@@ -497,8 +497,10 @@ iterate(){
   local RUNS=0
   local COMMAND="$*"
   local TMP_RET TMP_RET_CODE
+  set +e
   TMP_RET="$(${COMMAND})"
```

Comment:
This is also horrible way of checking the success of a command, when we care only about 0 or non-zero.